src/jsonRPCClient.php

```diff
@@ -9,172 +9,78 @@
  * http://implix.com
  */
```

Comment:
We might want to adapt the header pre-comment. Feel free to add "Monero Integrations Contributors"